plugin/sampling/strategystore/adaptive/processor_test.go

```diff
@@ -69,18 +71,29 @@ var (
 			interval: 60 * time.Second,
 		},
 	}
+}
 
-	errTestStorage = errors.New("storage error")
+func errTestStorage() error {
+	return errors.New("storage error")
+}
 
-	testCalculator = calculationstrategy.CalculateFunc(func(targetQPS, qps, oldProbability float64) float64 {
+func testCalculator() calculationstrategy.ProbabilityCalculator {
+	return calculationstrategy.CalculateFunc(func(targetQPS, qps, oldProbability float64) float64 {
 		factor := targetQPS / qps
 		return oldProbability * factor
 	})
-)
+}
+
+func TestAggregateThroughputInputsImmutability(t *testing.T) {
```

Comment:
if you run this test method on `master` branch it will fail

Reply:
+1

Given that the CI still fails due to some other issues, do you want to merge this PR without the `-shuffle=on` in the Makefile?

Reply:
I think it will profitable to keep the interesting commenting history therefore:

https://github.com/jaegertracing/jaeger/pull/3360